docs/developing_recipes_locally.md

```diff
@@ -27,12 +27,31 @@ This file is intended to provide all the commands you need to run in order to be
 
 ## Setup your environment
 
-1. Install a C++ development toolchain - ConanCenter's [build images](#testing-more-environments) are available
-2. [Install the Conan client](https://docs.conan.io/2/installation.html) - make sure to keep it up to date!
-3. Install CMake - this is the only tool which is assumed to be present
-   [see FAQ](faqs.md#why-recipes-that-use-build-tools-like-cmake-that-have-packages-in-conan-center-do-not-use-it-as-a-build-require-by-default) for details.
+In Conan Center, we require a baseline of tools and libraries that are assumed to be preinstalled on your system. This primarily affects Linux and similar platforms (FreeBSD, MinGW, etc.), while on Windows and macOS the distinction between system and non-system components is clearer.
```

Comment:
I would move the new changes to the `consuming recipes`  personally, in the sense that this _does_ affect anyone doing a `--build=missing` from a consumer point of view - and also when consuming there usually _is_ a compiler toolchain on the other end (their own consumer project, or their own recipe). I think the "developing recipes locally" is a different scope altogether - if sufficiently documented by the Conan client, I dont think there's a need for that section to be here either